docker/deprecated_images.json

```diff
@@ -0,0 +1 @@
1451Z"}, {"image_name": "demisto/feed-parser", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:04.632346Z"}, {"image_name": "demisto/geopy", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:06.659471Z"}, {"image_name": "demisto/gql", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:08.557290Z"}, {"image_name": "demisto/imap", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:10.588141Z"}, {"image_name": "demisto/intezer", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:12.717422Z"}, {"image_name": "demisto/jmespath", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:14.794983Z"}, {"image_name": "demisto/langdetect", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:16.844837Z"}, {"image_name": "demisto/ldap", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:19.112555Z"}, {"image_name": "demisto/minio", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:21.205968Z"}, {"image_name": "demisto/paho-mqtt", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:23.403448Z"}, {"image_name": "demisto/pyjarm", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:25.666241Z"}, {"image_name": "demisto/pymongo", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:27.903645Z"}, {"image_name": "demisto/python3-arrow", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:30.317140Z"}, {"image_name": "demisto/python_zipfile", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:32.810152Z"}, {"image_name": "demisto/sendgrid", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:35.169049Z"}, {"image_name": "demisto/stix2", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:37.529520Z"}, {"image_name": "demisto/stomp", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:39.917833Z"}, {"image_name": "demisto/tld", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:42.685435Z"}, {"image_name": "demisto/tweepy", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:45.278491Z"}, {"image_name": "demisto/vertica", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:47.771983Z"}, {"image_name": "demisto/xlrd-py3", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:50.253386Z"}, {"image_name": "demisto/zabbix", "reason": "docker image was merged to py3-tools", "created_time_utc": "2022-05-31T13:04:52.757590Z"}]
```

Comment:
@wissamg please fix your script so this file is formatted (with new lines) so it is readable.

Reply:
fixed

Reply:
Thanks a lot @Shellyber